meson.build

```diff
@@ -0,0 +1,108 @@
+project('wayland-eglstream', 'c',
+	version : '1.0.2',
+	default_options : [
+          'buildtype=debugoptimized',
+          'c_std=gnu99',
+          'warning_level=1',
+        ],
+	license : 'MIT',
+        meson_version : '>= 0.43'
+)
+
+cc = meson.get_compiler('c')
+
+wayland_eglstream_version = meson.project_version()
+ver_arr = wayland_eglstream_version.split('.')
+
+wayland_eglstream_major_version = ver_arr[0]
+wayland_eglstream_minor_version = ver_arr[1]
+wayland_eglstream_micro_version = ver_arr[2]
+
+eglexternalplatform = dependency('eglexternalplatform', version : ['>=1.0', '<2'])
+wayland_server = dependency('wayland-server')
+wayland_client = dependency('wayland-client')
+
+wl_scanner = find_program('wayland-scanner')
+
+inc = include_directories(
+    'include',
+    'wayland-egl',
+    'wayland-eglstream',
+)
+
+top_srcdir_inc = include_directories('.')
+
+cdata = configuration_data()
+cdata.set_quoted('PACKAGE',            meson.project_name(),                                            description : 'Name of package')
+cdata.set_quoted('PACKAGE_BUGREPORT',  'mvicomoya@nvidia.com',                                          description : 'Define to the address where bug reports for this package should be sent.')
+cdata.set_quoted('PACKAGE_NAME',       meson.project_name(),                                            description : 'Define to the full name of this package.')
+cdata.set_quoted('PACKAGE_STRING',     '@0@ @1@'.format(meson.project_name(), meson.project_version()), description : 'Define to the full name and version of this package.')
+cdata.set_quoted('PACKAGE_TARNAME',     meson.project_name(),                                           description : 'Define to the one symbol short name of this package.')
+cdata.set_quoted('PACKAGE_URL',        '',                                                              description : 'Define to the home page for this package.')
+cdata.set_quoted('PACKAGE_VERSION',    meson.project_version(),                                         description : 'Define to the version of this package.')
+
+check_headers = [
```

Comment:
I guess this is a question for @mvicomoya but it doesn't seem like these headers are needed. I wonder if we can remove check_headers and check_functions completely.

Reply:
Yeah, why do we need these checks?

Reply:
I added this because I was not sure if this is required.